From 98d079a828fafd1e15dfb7e2136fbcf29e785a14 Mon Sep 17 00:00:00 2001
From: jinghua <xigu@marvell.com>
Date: Thu, 8 Sep 2016 22:47:17 +0800
Subject: [PATCH 0948/2241] dts: a3700: update usb3 vbus gpio pin number for
 rev2 devel-board

- For Armada3700 rev1 devel-board, IO expander is used as vbus
  of usb32. But for Armada3700 rev2 devel-board, it has been
  changed to South Bridge GPIO0.

Change-Id: I99029077deac150885bc0c864679a89ef34ec6bc
Signed-off-by: jinghua <xigu@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/32635
Reviewed-by: Wilson Ding <dingwei@marvell.com>
Tested-by: Star_Automation <star@marvell.com>
---
 arch/arm64/boot/dts/marvell/armada-3720-db.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-3720-db.dts b/arch/arm64/boot/dts/marvell/armada-3720-db.dts
index 00f89bb..3830bd8 100644
--- a/arch/arm64/boot/dts/marvell/armada-3720-db.dts
+++ b/arch/arm64/boot/dts/marvell/armada-3720-db.dts
@@ -225,7 +225,7 @@
 		regulator-max-microvolt = <5000000>;
 		enable-active-high;
 		regulator-always-on;
-		gpio = <&expander0 1 GPIO_ACTIVE_HIGH>;
+		gpios = <&gpio_sb 0 GPIO_ACTIVE_HIGH>;
 	};
 };
 
-- 
2.7.4

